iron-menu-behavior.js

```diff
@@ -138,6 +139,7 @@ export const IronMenuBehaviorImpl = {
 
     this.items.forEach(function(item) {
       item.setAttribute('tabindex', item === selectedItem ? '0' : '-1');
+      this._applySelection(item, item === selectedItem);
```

Comment:
`IronSelectableBehavior._applySelection`, which `this._applySelection` calls into, will dispatch an `iron-select` and `iron-deselect` event for the given item. So, I think this means that doing this in `_resetTabindices` will dispatch events for all items, which I'm not sure we want to do since that happens every time items change or the element is attached. Maybe directly setting `aria-selected` here is more appropriate? WDYT?

Reply:
Yeah, you're right. I changed the code to directly set aria-selected in _resetTabindices instead.